lib/msal-node/src/client/PublicClientApplication.ts

```diff
@@ -89,15 +104,36 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * Acquires a token by requesting an Authorization code then exchanging it for a token.
      */
     async acquireTokenInteractive(request: InteractiveRequest): Promise<AuthenticationResult> {
+        const correlationId = request.correlationId || this.cryptoProvider.createNewGuid();
+        this.logger.trace("acquireTokenInteractive called", correlationId);
+        const { openBrowser, successTemplate, errorTemplate, windowHandle, ...remainingProperties } = request;
+
+        if (this.nativeBrokerPlugin) {
+            const brokerRequest: NativeRequest = {
+                ...remainingProperties,
+                clientId: this.config.auth.clientId,
+                scopes: request.scopes || OIDC_DEFAULT_SCOPES,
```

Comment:
This is an OR condition, defaults are used if no scopes were requested explicitly and this keeps the behavior consistent with our non-brokered flows.

Reply:
Same question whether it's a conditional or a or'ing the collections together.  Do we really need to add these, or should we just user the broker defaults here if none are specified?  How does us not leaving this empty impact what the broker does?

Reply:
2 reasons to do this:
1. Broker requests _require_ a value for scopes, our APIs do not
2. We provide defaults in non-brokered flows and our APIs should behave the same whether the broker is used or not. Relying on the broker's defaults instead of our own may result in different behaviors on different platforms